variant-box/pi-setup.sh

```diff
@@ -75,7 +75,65 @@ sudo zip -j   -u /boot/secrets.zip   /var/lib/tor/ssh/hostname
 
 
 # TODO: detect `-source` suffix in `bitcoind-version`; if present build from source, if not build from binaries
-# TODO: how to detect arch for binaries?
+BITCOINVER=`cat bundle/bitcoind-version`
+if [[ $BITCOINVER == *"source"* ]]; then 
+    echo "Building bitcoin from source"
```

Comment:
Building from sources is literally just few lines below - can you just uncomment and copy it into this if?

Reply:
sure